evm/src/cpu/kernel/asm/secp256k1/lift_x.asm

```diff
@@ -0,0 +1,825 @@
+// Returns y such that (x,y) is on Secp256k1 and y&1 = v - 27,
```

Comment:
Hm I think we'll also want to use this when verifying newer transactions which don't use the `v = 27 + y_parity` format. Might be best to take `y_parity` as a bool here, and move `v - 27` to some code that's specifically for the ECREC precompile? Or we can tweak it later when we add transaction verification.

Reply:
Good point, done.